packages/core/package.json

```diff
@@ -211,13 +99,14 @@
   },
   "scripts": {
     "lint": "eslint .",
-    "test": "jest",
-    "build": "rm -rf ./dist && NODE_OPTIONS=\"--max-old-space-size=12288\" bunchee",
+    "build": "rm -rf ./dist && pnpm run build:esm && pnpm run build:cjs && pnpm run build:type",
+    "build:esm": "swc src -d dist --strip-leading-paths --config-file .swcrc",
```

Comment:
If i read this correctly, you're using swc to generate ESM/CJS code and tsc to emit declarations only.

Why this combination? What are alternatives?

Reply:
swc could also support polyfill and generate to different target if we want. TSC only generate based on tsconfig and   I don't think add multiple tsconfigs is a good choice